### DISCUSSION TITLE
Create a DAO (Decentralized Anonymous Organization) so freely/anonymously vote and work on projects

### DISCUSSION THREAD

**Discussion by mrnobud** [WeAreMahsaAmini/Core#225] [Ideas]
Create a DAO so people can freely/anonymously join and participate in the community, vote or contribute. We can even start fundraising, make sure funds are secured through smart contracts and only used for purposes that the community wants. e.g sponsoring an ad in time square.  We can then start voting for major movements, ides, etc. 

**All rules will be coded in smart contacts, everything would be on blockchain, verifiable, and secure.** 

- This can be done via an NFT project, which is simple to build. 
- We can distribute a min number of initial set to project team, and/or do an invitation only limited number of NFTs, we can do a second run and give an NFT to anyone who contributes a certain way (this will be done via smart contracts and everything will be transparent). For example, if you have a twitter account with some number of hashtags, you will get one, etc. 
- We can also sell some, if we want to fund raise, and each NFT can be different and hold different voting power. 
- We can make initial or all NFTs non-transferable, so it only be used to participate in the DAO and for no financial gain. 

Project goals:

The world's first blockchain token related to supporting a revolution
Fundraising for protesters in Iran
Introducing Iran's revolution and protests to all the people of the world and cryptocurrency activists
Spiritual value
Financial value for investors and supporters
Using a decentralized system for financial assistance

**Comment by amirnazari96** [comment]
I would suggest, as a Free Iran, we could have a part like the Github discussions, so then people can post their ideas, and the idea that gets the most votes will execute. 
So we can start the first free decentralized government!
